packages/elements-core/src/components/Docs/HttpService/AdditionalInfo.tsx

```diff
@@ -1,16 +1,21 @@
-import { Panel } from '@stoplight/mosaic';
+import { NodeAnnotation, Panel } from '@stoplight/mosaic';
 import { IHttpService } from '@stoplight/types';
 import React from 'react';
 
+import { useOptionsCtx } from '../../../context/Options';
 import { MarkdownViewer } from '../../MarkdownViewer';
 
 interface AdditionalInfoProps {
+  id?: string;
   termsOfService?: string;
   contact?: IHttpService['contact'];
   license?: IHttpService['license'];
 }
 
-export const AdditionalInfo: React.FC<AdditionalInfoProps> = ({ termsOfService, contact, license }) => {
+export const AdditionalInfo: React.FC<AdditionalInfoProps> = ({ id, termsOfService, contact, license }) => {
+  const { nodeHasChanged } = useOptionsCtx();
+  const hasChanged = nodeHasChanged?.({ nodeId: id || '', attr: ['termsOfService', 'contact', 'license'] });
```

Comment:
I'm a bit on the edge with it. Let's perhaps make `id` required?
I assume the `id` we pass here comes from `IHttpService`, so it should be always there.